src/cozmo/robot.py

```diff
@@ -1666,14 +1662,15 @@ def go_to_object(self, target_object, distance_from_object,
         return action
 
     def dock_with_cube(self, target_object, approach_angle=None,
-                       alignment_type=None, distance_from_marker=None,
+                       alignment_type=robot_alignment.RobotAlignmentTypes.LiftPlate,
+                       distance_from_marker=None,
                        in_parallel=False, num_retries=0):
         '''Tells Cozmo to dock with a specified cube object.
 
         Args:
-            target_object (:class:`cozmo.objects.LightCube`): The destination object.
-            approach_angle (:class:`cozmo.util.Angle`): The angle to approach the cube from.
-            alignment_type (:class:`cozmo.robot_alignment.RobotAlignmentTypes`): which part of the robot to line up with the object
+            target_object (:class:`cozmo.objects.LightCube`): The cube to dock with.
+            approach_angle (:class:`cozmo.util.Angle`): The angle to approach the cube from.  For example, 180 will cause cozmo to drive past the cube and approach it from behind.
```

Comment:
"180" is ambiguous here as the angle type doesn't define the units (could be degrees or radians depending on how they called it), so suggest changing the doc here to say "180 degrees"